rebasehelper/cli.py

```diff
@@ -163,8 +174,8 @@ class CliHelper(object):
 
     @staticmethod
     def run():
+        debug_log_file = None
         try:
-            debug_log_file = None
             # be verbose until debug_log_file is created
             handler = LoggerHelper.add_stream_handler(logger, logging.DEBUG)
             cli = CLI()
```

Comment:
This is intentional, since `sys.argv[1:]` is what `ArgumentParser.parse_args` uses if not provided with any arguments.